pkg/planner/core/exhaust_physical_plans.go

```diff
@@ -1504,16 +1504,29 @@ func constructIndexJoinInnerSideTaskWithAggCheck(p *logicalop.LogicalJoin, prop
 				physicalIndexScan, _ = dsCopTask.indexPlan.Children()[0].(*PhysicalIndexScan)
 			}
 			if physicalIndexScan != nil {
+				stats := la.StatsInfo()
```

Comment:
Please reduce the duplication of code. Calculate stats only once.

Reply:
I have merged the two parts of the code.